src/metabase/native_query_analyzer.clj

```diff
@@ -16,7 +16,7 @@
    [macaw.core :as macaw]
    [metabase.config :as config]
    [metabase.driver :as driver]
-   [metabase.driver.util :as driver.u]
+   [metabase.lib.native :as lib.native]
```

Comment:
We still depend on both these modules, but `metabase.lib` is the higher-level one.